.github/workflows/make_bundle.yml

```diff
@@ -50,6 +50,9 @@ jobs:
           python -m pip install briefcase==0.3.1 tomlkit wheel dmgbuild>=1.4.2
           python -m pip install PySide2==5.15.2
           python -m pip install -e .
+      - name: Install latest briefcase for macos
```

Comment:
confirmed latest briefcase fixed the naming issue in menubar, however the latest should ONLY be applied to macos, there is some regression bugs between dmgbuild and briefcase that would cause the import to crash due to https://github.com/al45tair/dmgbuild/blob/master/dmgbuild/core.py#L57 failes on non-mac environments

Reply:
to be more specific, on macos the platform.mac_ver()[0].split('.') resulted in '' therefore int(v) raises value error, and because this is at import level, any package trying to import it would fail. We would have to patch the code at file level (similar to what we are doing now to insert the extra sleep) to skip this value assignment, so that the import would work. I tried this approach and it is working, but too much changes so I didn't bother include in this PR

Reply:
just curious, did you try the updated briefcase on other platforms, and just not include dmgbuild there (since it should only be necessary on macos anyway)

Reply:
The briefcase fails on linux for the qt library linking still, so I didn't bother try more things, but theoretically I think this is a great idea

Reply:
ok!  thanks for checking

Reply:
do you happen to know if we can update windows?  Since linux is the least used bundle... might be nice to just special case linux, rather than mac?

Reply:
I am afraid I don't have time this week (I am packing up and moving). Feel free to create an issue to track the thought If you would like to.